utils/firestore.js

```diff
@@ -2125,6 +2125,53 @@ const updateUserPhoneSigninMethod = async (phone, uid) => {
     }
 }
 
+/**
+ * queryDailyReportParticipants
+ * @param {}
+ * returns participants that are no more than 2 days old from the time of check in date/time
+ * Using promises to resolve array of objects
+ * object containes essential information Collection Location (951355211), Connect ID,
+ * F+L Name (996038075 & 399159511), Check-In (840048338), Collection ID (820476880),
+ * Collection Finalized(556788178), Check-Out (343048998)
+ * Queried from Participants & Biospecimen table
+ */
+
+const queryDailyReportParticipants = async () => {
+    const twoDaysAgo = new Date(new Date().getTime() - (2 * 24 * 60 * 60 * 1000)).toISOString();
```

Comment:
Comment only:
This is great for a rolling 48-hour window. Might be worth hardcoding the calculation (micro-optimization).
If, in the future, there's a preference for calendar days as the reporting window, something like this should work:
```javascript
const twoDaysAgo = new Date();
twoDaysAgo.setDate(twoDaysAgo.getDate() - 2);
twoDaysAgo.setHours(0, 0, 0, 0);
const twoDaysAgoIso = twoDaysAgo.toISOString();
```